corehq/apps/export/dbaccessors.py

```diff
@@ -130,23 +130,37 @@ def _get_export_instance(cls, key):
 
 def get_all_daily_saved_export_instance_ids(accessed_after=None):
     """
-    get all saved exports accessed after the timestamp
-    :param accessed_after: reports that have been accessed after this time
+    get all saved exports or accessed after the timestamp
+    :param accessed_after: datetime to get reports that have been accessed after this timestamp
     """
     from .models import ExportInstance
-    results = ExportInstance.get_db().view(
-        "export_instances_by_is_daily_saved/view",
-        include_docs=False,
-        reduce=False,
-    ).all()
-    if accessed_after:
-        # return exports that have accessed_after set after the cutoff requested
-        # or don't have any value set for it yet
-        return [result['id'] for result in results
-                if not result['key'][0] or
-                string_to_utc_datetime(result['key'][0]) >= accessed_after]
+    if not accessed_after:
```

Comment:
FYI it looks like this function is used in two places, the task and a test.  The task always passes in a datetime, but the test doesn't.  I'd make `accessed_after` a required argument and update the test accordingly, that way you can remove a fork in the view code, and to be sure that the test tests the functionality that's used in normal operation.

Reply:
Sure Ethan. Added.

Reply:
Read that too quick. https://github.com/dimagi/commcare-hq/pull/22459